### ISSUE TITLE
uiowa.edu accessibility issues (as of 12/18/25)

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#9489]
Siteimprove flagged issues/potential issues of note:
- ~(AA) Heading is not descriptive -  only issue flagged in compliance report, but is a~ false positive 
- (AA) Is it clear which page element has focus from the keyboard? - this potential issue is backed up by manual testing. Basically, on pages using the after content region, like https://uiowa.edu/explore/biomedical-engineering-degree/which-college-is-best-for-biomedical-engineering, the keyboard focus indicator can disappear behind the region. 
- ~(A) Does video without audio have an accessible alternative? - another potential issue backed up by manual testing. The banner video at the top of the homepage has no audio and so no captions. Ideally there would be an accessible description of it somewhere, or perhaps mark it as decorative?~ This should be resolved by https://github.com/uiowa/uiowa/pull/9536

Manual testing issues: 
- Skipped heading level so page does not start with a h1 - Siteimprove also flagged this as a best practice issue
- some of the tracking pixels are not marked as decorative and thus getting flagged by the alt text checker - is this a content team, not dev team issue?
  - ```<img src="https://t.co/1/i/adsct?bci=4&amp;dv=America%2FChicago%26en-US%2Cen%26Google%20Inc.%26MacIntel%26127%261920%261200%2610%2624%261920%261175%260%26na&amp;eci=3&amp;event=%7B%7D&amp;event_id=fbbf5cc5-3d78-4d59-a000-28eee1f22e0d&amp;integration=gtm&amp;p_id=Twitter&amp;p_user_id=0&amp;pl_id=83d2e082-aab3-4714-be96-832a9f0b34c3&amp;pt=The%20University%20of%20Iowa&amp;tw_document_href=https%3A%2F%2Fuiowa.edu%2F&amp;tw_iframe_status=0&amp;txn_id=nw1cb&amp;type=javascript&amp;version=2.3.35" height="1" width="1" style="display: none;" class="wai-bad">```
  - ```<img src="https://analytics.twitter.com/1/i/adsct?bci=4&amp;dv=America%2FChicago%26en-US%2Cen%26Google%20Inc.%26MacIntel%26127%261920%261200%2610%2624%261920%261175%260%26na&amp;eci=3&amp;event=%7B%7D&amp;event_id=fbbf5cc5-3d78-4d59-a000-28eee1f22e0d&amp;integration=gtm&amp;p_id=Twitter&amp;p_user_id=0&amp;pl_id=83d2e082-aab3-4714-be96-832a9f0b34c3&amp;pt=The%20University%20of%20Iowa&amp;tw_document_href=https%3A%2F%2Fuiowa.edu%2F&amp;tw_iframe_status=0&amp;txn_id=nw1cb&amp;type=javascript&amp;version=2.3.35" height="1" width="1" style="display: none;" class="wai-bad">```

# Scope
- [ ] (AA) Is it clear which page element has focus from the keyboard?
- [ ] (A) Missing alt text on two spacer images related to tracking pixels
- [ ] (Best Practice) Skipped heading level so page does not start with a h1?

**Comment by briand44** [comment]
@bspeare I think this is the potential issue in Siteimprove about the focus - https://my2.siteimprove.com/Accessibility/995976/NextGen/Issue/1?ruleId=65&pageSegments=&issueKind=2&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

**Comment by bspeare** [comment]
Lydia is going to disable the Twitter scripts that are delivering the img pixels.   They no longer need them.
